.github/workflows/markdown-link-check.yml

```
@@ -0,0 +1,12 @@
+name: Check Markdown links
+
+on: push
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

main is the default branch, https://github.com/actions/checkout

```suggestion
    - uses: actions/checkout@main
```